django_x509/base/admin.py

```diff
@@ -175,6 +186,16 @@ def revoke_action(self, request, queryset):
 
     revoke_action.short_description = _('Revoke selected certificates')
 
+    def renew_cert(self, request, queryset):
+        renewed_rows = 0
+        for cert in queryset:
+            cert.ca.renew_ca()
+            renewed_rows += 1
+        message = _('{0} Certificates have been successfully renewed'.format(renewed_rows))
```

Comment:
please use pluralization to display the correct message depending on the amount of objects on which the operation was performed, see here: https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#pluralization

Follow the exact style used in that example.